library/Garp/Cli/Command/Cdn.php

```diff
@@ -93,6 +108,71 @@ public function help() {
 
     }
 
+    /**
+     * Take the required variables necessary for distributing assets.
+     *
+     * @return array
+     */
+    protected function _gatherConfigVars() {
+        $source = $this->_stdin ?
+            $this->_parseStdin($this->_stdin) :
+            Zend_Registry::get('config')->toArray();
+        $cdnConfig = f\prop('cdn', $source);
+        $s3Config = f\prop('s3', $cdnConfig);
+        return array(
+            'apikey'          => f\prop('apikey', $s3Config),
+            'secret'          => f\prop('secret', $s3Config),
+            'bucket'          => f\prop('bucket', $s3Config),
+            'readonly'        => f\prop('readonly', $cdnConfig),
+            'gzip'            => f\prop('gzip', $cdnConfig),
+            'gzip_exceptions' => f\prop('gzip_exceptions', $cdnConfig)
+        );
+    }
+
+    /**
+     * Parse STDIN as JSON.
+     * Output is formatted like the standard assets.ini.
+     *
+     * @param string $stdin
+     * @return array
+     */
+    protected function _parseStdin($stdin) {
+        try {
+            $values = Zend_Json::decode($stdin);
+            // Switch ENV vars to the given values.
+            $currentEnv = $this->_updateEnvVars($values);
+            $config = new Zend_Config_Ini(APPLICATION_PATH . '/configs/assets.ini', 'production');
```

Comment:
Can we please add a small note here about why 'production' needs to be explicit?